docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,315 @@
+Implement minimal example of the ``pl.LightningModule`` and simple ``DataLoader``.
+
+.. code-block:: python3
 
+    # pl.LightningModule
+    class LitModel(pl.LightningModule):
         def __init__(self):
-            ...
+            super().__init__()
+            self.l1 = torch.nn.Linear(28 * 28, 10)
 
         def forward(self, x):
-            ...
+            return torch.relu(self.l1(x.view(x.size(0), -1)))
 
         def training_step(self, batch, batch_idx):
-            ...
+            x, y = batch
+            y_hat = self(x)
+            loss = F.cross_entropy(y_hat, y)
+            result = pl.TrainResult(minimize=loss)
+            result.log('train_loss', loss)
+            return result
 
         def configure_optimizers(self):
-            ...
+            return torch.optim.Adam(self.parameters(), lr=PARAMS['learning_rate'])
+
+    # DataLoader
+    train_loader = DataLoader(MNIST(os.getcwd(), download=True, transform=transforms.ToTensor()),
+                              batch_size=PARAMS['batch_size'])
 
-        @pl.data_loader
-        def train_dataloader(self):
-            ...
+Cross Entropy loss in the ``training_step`` method will be logged to Neptune in every training step. ``DataLoader`` is a purely PyTorch object. Notice, that you pass ``learning_rate`` and ``batch_size`` from the ``PARAMS`` dictionary - all params will be logged as experiment parameters.
 
-        @pl.data_loader
-        def val_dataloader(self):
-            ...
+Step 4: Create NeptuneLogger
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Instantiate ``NeptuneLogger`` with necessary parameters.
 
-Create the **NeptuneLogger** with all the information you want to track
------------------------------------------------------------------------
 .. code-block:: python3
 
     from pytorch_lightning.loggers.neptune import NeptuneLogger
 
     neptune_logger = NeptuneLogger(
         api_key="ANONYMOUS",
         project_name="shared/pytorch-lightning-integration",
-        experiment_name="default",  # Optional,
-        params={"max_epochs": 10,
-                "batch_size": 32},  # Optional,
-        tags=["pytorch-lightning", "mlp"]  # Optional,
-    )
-
-Create the **Trainer** and pass **neptune_logger** to logger
-------------------------------------------------------------
-.. code-block:: python3
-
-    from pytorch_lightning import Trainer
+        params=PARAMS)
 
-    model = CoolSystem()
-    trainer = Trainer(max_epochs=10, logger=neptune_logger)
+``NeptuneLogger`` is an object that integrates Neptune with PyTorch Lightning allowing you to track experiments. It's a part of the lightning library. In this minimalist example we use public user `"neptuner"`, who has public token: `"ANONYMOUS"`.
 
-    trainer.fit(model)
+.. tip::
 
-Log additional information after the **.fit** loop ends
--------------------------------------------------------
+    Make sure to use your API token in your projects. Read more about how to |token|.
```

Comment:
I don't know if it should be make sure or something like if you want to use in your project. 

Some people may just want to run the how-to and there is no must there

Reply:
corrected.